variables.tf

```diff
@@ -29,9 +29,12 @@ variable "node_count" {
   }
 }
 
-variable "proxy_ip" {
+variable "http_proxy_uri_authority" {
```

Comment:
This will likely cause a refactor in the [tfe_init](https://github.com/hashicorp/terraform-random-tfe-utility/blob/main/modules/tfe_init/templates/tfe.sh.tpl#L32) code, which is good, just noting.